runtime-transaction/src/transaction_meta.rs

```diff
@@ -25,7 +25,9 @@ pub trait StaticMeta {
 /// have both meta data populated and available.
 /// Dynamic metadata available after accounts addresses are loaded from
 /// on-chain ALT, examples are: transaction usage costs, nonce account.
-pub trait DynamicMeta: StaticMeta {}
+pub trait DynamicMeta: StaticMeta {
+    fn i_am_dynamic(&self) -> bool;
```

Comment:
🤔 not sure this is necessary, since we can't really test that types do **not** have access.
I would prefer this is just left empty until we actually have dynamic metadata, wdyt?

Reply:
no logic should depend on a property like this. the point of type safety is that only one type is even valid in the given context.

Reply:
that's only used for trying and confirming different methods, I forgot to remove it. Thanks for pointing out. Cleanup at cbd3193